build.sbt

```diff
@@ -26,3 +26,37 @@ Global / onLoad := (Global / onLoad).value.andThen { s =>
   dynverAssertTagVersion.value
   s
 }
+
+lazy val scala212 = "2.12.19"
+ThisBuild / crossScalaVersions := Seq(scala212)
+ThisBuild / scalaVersion := scala212
+
+ThisBuild / githubWorkflowBuild := Seq(WorkflowStep.Sbt(List("test", "scripted")))
+
+ThisBuild / githubWorkflowTargetTags ++= Seq("*.*.*")
+ThisBuild / githubWorkflowPublishTargetBranches :=
+  Seq(
+    RefPredicate.Contains(Ref.Tag("*.*.*")),
```

Comment:
According to https://docs.github.com/en/actions/learn-github-actions/expressions#contains the `*` pattern should be supported